File: Herencia/ejercicio 1/Python/Vehiculo.py
class Vehiculo:
    def __init__(self, marca, modelo, año, precio):
        self.marca = marca
        self.modelo = modelo
        self.año = año
        self.precio = precio
    def mostrar(self):
        return "Marca: ", self.marca, "Modelo: ", self.modelo, "Año: ", self.año, "Precio Base: ", self.precio 

File: Herencia/ejercicio 1/Java/Vehiculo.java
public class Vehiculo {
    public String marca;
    public String modelo;
    public int año;
    public int precio;

    public Vehiculo(String marca, String modelo, int año, int precio){
        this.marca = marca;
        this.modelo = modelo;
        this.año = año;
        this.precio = precio;
    }
    public String mostrar(){
        return "marca: " + marca + "modelo: " + modelo + "año: " + año + "precio: " + precio;
    }
}
